plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -1471,6 +1474,9 @@ function( $subscription ) use ( $site_id ) {
 				$local                              = array_shift( $local );
 				$subscription['local']['installed'] = true;
 				$subscription['local']['version']   = $local['Version'];
+				$subscription['local']['type']      = $local['_type'];
+				$subscription['local']['path']      = $local['_filename'];
+				$subscription['local']['slug']      = null;
```

Comment:
Locally, `props.subscription.local.slug` values for all of my subscriptions are `null`. I think this prevents updates now, with error **no_theme_specified**

![image](https://github.com/woocommerce/woocommerce/assets/10389957/1231099c-2953-4079-ba3f-4d960c2a2b72).

Could it be because of this line?

Reply:
I added a fix that will prevent the button from working if there is data missing. 
I'm unable to recreate it locally and it looks to me like this is old cache before the API had slug data.